### DISCUSSION TITLE
Can't reverify my academic affiliation.

### DISCUSSION THREAD

**Discussion by Akasaz** [community/community#84940] [GitHub Education]
### Hi everyone,

Since last November, when I visit the webpage 'https://education.github.com/benefits', it prompts me to' please submit your information again.'
![image](https://github.com/community/community/assets/127120939/10772914-7f82-4c23-957b-aa5c200df15c)

But when I click on the blue text, it automatically jumps to' https://github.com/pricing', so that I can't submit my information.
![image](https://github.com/community/community/assets/127120939/55450f08-5985-408c-b680-b84e2a3dde16)

And I also tried to click "Reverify your academic affiliation" on the menu bar in the upper right corner. It still jumped to the same website.
![image](https://github.com/community/community/assets/127120939/bba49db4-f4a1-4e85-97e6-05437b0582ff)

Based on the suggestions I found in the community discussions, I have tried updating my account information, reconfiguring 2FA, and submitting a support ticket. 

I also attempted some risky actions. I unlinked my educational email and registered a new account with it. Then, I used the new account for student verification. However, it prompted me that I couldn't verify an already verified student account. As a result, I deleted that account and re-linked it to my current account.

**Comment by Peter-K-Joseph** [comment]
You aren't alone bud..
![image](https://github.com/community/community/assets/69970683/db84110e-b3da-4832-8b42-271127df4696)
Im having the same issue.

**Reply by liuliangsir** [reply]
Same issue here. In addition, why could you take a screenshot from [re-verification](https://education.github.com/discount_requests/application) page since I will be redirected [pricing](https://github.com/pricing) page when accessing [re-verification](https://education.github.com/discount_requests/application) page.

**Comment by kmadaminjanova0508** [comment]
Hi @Akasaz I think in your situation you need to  reach out to Git hub support service
